wiki/Ranking_Criteria/osu!mania/es.md

```diff
@@ -0,0 +1,205 @@
+
+### Términos comunes
+
+- **BPM**: Tempo de una canción medido en pulsaciones por minuto.
+- **Column**: Líneas verticales en el campo de juego donde las notas son colocadas. El editor usa `0|1|2|3|4|5|6|7|8|9` para las etiquetas de marcas de tiempo.
+- **Key mode**: La cantidad de columnas presentes en una dificultad.
+- **Keysounding**: Aplicar hitsounds que son extremadamente similar o directamente de la canción en sus respectivos lugares. Este método de hitsounding generalmente proporciona poca retroalimentación al jugador.
+- **Slider velocity**: La velocidad a la que se mueven las notas/barras en el campo de juego. La velocidad base del slider (1.0x) se basa en la velocidad de desplazamiento del usuario y se pueden realizar cambios adicionales a través de puntos de tiempo heredados (verdes).
+- **Long-term slider velocity**:Un cambio de velocidad del slider que dura más de 4 compases.
+- **Musical phrase**: Una unidad independiente musical que suele ser de 4 compases de largo.
+- **Short-term slider velocity**: Un cambio en la velocidad del slider que dura menos que 1/2 de un pulso.
+
+### Patrones
+
+- **Anchor**: Un patrón que tiene una cantidad relativamente alta de notas en una columna comparada con las otras, usualmente se encuentran entre diferentes tipos de streams.
+- **Bracket**: Un patrón que involucra múltiples trills usando columnas cerca una de las otras.
+- **Chord**: Dos o más notas presionadas al mismo tiempo.
+- **Chordstream (Jumpstream, Handstream, Quadstream)**: Streams que mezclan diferentes tipos de chords.
+- **Grace**: Dos o más notas presionadas en sucesión rápida, generalmente usando snaps de 1/6 o más.
+- **Hand**: Tres notas presionadas al mismo tiempo. Ocasionalmente referida como una "triple".
+- **Jack**: Tres o más notas consecutivas en la misma columna, usualmente presionada a un ritmo más rápido que las notas aledañas.
+- **Jump**: Dos notas presionadas al mismo tiempo. Ocasionalmente referida como una "doble".
+- **Jumptrill**: Un trill consistiendo en dos jumps que alternan.
+- **Minijack**: Dos notas consecutivas en la misma columna, usualmente presionadas a un ritmo más rápido que las notas aledañas.
+- **Release**: El final de una nota larga el cual debe soltarse en una cierta ventana de tiempo para poder conseguir el puntaje completo apropiadamente.
+- **Roll**: Una variación del stream que consiste de notas que van desde la primera columna hasta lo ultima o desde la última hasta la primera, usualmente en sucesión rápida.
+- **Shield**: Una nota que aparece en la misma columna inmediatamente antes del inicio de una nota larga.
+- **Split-jumptrill**: Un jumptrill que requiere el uso de las dos manos al mismo tiempo para ser realizado correctamente (por ejemplo, dos trills de una mano).
+- **Stream**: Notas constantes con el mismo intervalo por al menos una pulsación que no se repite para crear un minijack.
+- **Trill**: Más de dos notas que alternan entre dos columnas sin interrupción.

```

Comment:
In those that exist, you could put the translation in parentheses or the other way around and change it in the rest of the article.

Reply:
I like this idea actually, but I think it'd be a little counterintuitive (mania speaking) to do it that way since most of the Latin American community, including Europeans ones as well use the English terms rather than the Spanish ones.